app/helpers/availabilities_helper.rb

```diff
@@ -2,19 +2,62 @@
 
 module AvailabilitiesHelper
   def generate_cell(day, time_range)
-    generate(@availabilities[[day, time_range.id]],
-             calc_colspan(time_range.start_time, time_range.end_time))
+    generate(@availabilities[[day, time_range.id]])
   end
 
-  def generate(availability, col_span)
+  def generate(availability)
     id = availability.id
-    td_attributes = { align: 'center', class: 'checkbox_cell',
-                      onclick: "toggle(#{id})", id: "cell_#{id}",
-                      colspan: col_span }
+    check_box_tag("availability_ids[#{id}]", id, availability.status,
+                  style: 'visibility: hidden;')
+  end
 
-    content_tag :td, td_attributes do
-      check_box_tag "availability_ids[#{id}]", id, availability.status,
-                    style: 'visibility: hidden; display: none:'
+  def stylise_user(user)
```

Comment:
Instead of this probably slapping the `.mc` CSS class is better so that we can style it anyway we want later on

Reply:
Done